### PR TITLE
Add navigation icons to psammead-assets

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #NUMBER

**Overall change:** 
Added hamburger and cross icons which will be used in new Navigation Iteration.

**Code changes:**
- Added svgs for hamburger and cross icons
- Added stories for navigation icons

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by FK78 — APPROVED** [review]
LGTM!

**Review by rhenshaw56 — APPROVED** [review]

**Review by DenisHdz — CHANGES_REQUESTED** [review]
If a Firefox user sets the background to white in High Contrast mode, the icon won't be visible.
Can you replace it with currentColor as we do with the MediaIcons?

```
import React from 'react';
import styled from 'styled-components';

const MenuIcon = styled.svg`
  color: #FFF;
  fill: currentColor;
`;

const navigationIcons = {
  hamburger: (
    <MenuIcon width={44} height={44}>
      <path
        d="M12 29h21v-2.333H12V29zm0-5.833h21v-2.334H12v2.334zM12 15v2.333h21V15H12z"

      />
    </MenuIcon>
  ),
  cross: (
    <MenuIcon width={44} height={44}>
      <path
        d="M26.741 15L21.6 20.142 16.458 15 15 16.458l5.142 5.142L15 26.742l1.458 1.458 5.142-5.141 5.141 5.141 1.459-1.458-5.142-5.142 5.142-5.142z"
        fillRule="evenodd"
      />
    </MenuIcon>
  ),
};

export default navigationIcons;
```

**Review by DenisHdz — APPROVED** [review]
LGTM 💯

File: packages/utilities/psammead-assets/src/svgs/navigationIcons.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import styled from 'styled-components';
+
+const MenuIcon = styled.svg`
+  color: #fff;
+  fill: currentColor;
+`;
+
+const navigationIcons = {
+  hamburger: (
+    <MenuIcon width={44} height={44}>
```
Comment (sareh) [inline packages/utilities/psammead-assets/src/svgs/navigationIcons.jsx]: Width and height properties should have units associated with them.
```suggestion
    <MenuIcon width="44px" height="44px">
```

File: packages/utilities/psammead-assets/src/svgs/navigationIcons.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      <path d="M12 29h21v-2.333H12V29zm0-5.833h21v-2.334H12v2.334zM12 15v2.333h21V15H12z" />
+    </MenuIcon>
+  ),
+  cross: (
+    <MenuIcon width={44} height={44}>
```
Comment (sareh) [inline packages/utilities/psammead-assets/src/svgs/navigationIcons.jsx]:
```suggestion
    <MenuIcon width="44px" height="44px">
```

**Review by sareh — CHANGES_REQUESTED** [review]
Looks great. Just one point regarding width/height values.

**Review by sareh — APPROVED** [review]
👍 Looks great.

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**